contrib/util/express.php

```diff
@@ -46,126 +65,159 @@
  
     //setup of database
 
-    echo "Connecting to MySQL Server...\n";
+    echo "Connecting to MySQL Server...
```

Comment:
agreed. My bad that it slipped in there. I'm actually confused why it didn't get picked up by the syntax linter, since I see some errors. I'll push a quick fix.